enterprise/server/cmd/executor/executor.go

```diff
@@ -86,6 +86,8 @@ func InitializeCacheClientsOrDie(cacheTarget string, realEnv *real_environment.R
 	} else {
 		if cacheTarget == "" {
 			log.Fatalf("No cache target was set. Run a local cache or specify one in the config")
+		} else if cacheTarget == "grpcs://cloud.buildbuddy.io" {
```

Comment:
do we also allow specifying this as `cloud.buildbuddy.io`? wonder if we should check for that here too

Reply:
I'm sorry, I don't think I understand. Check for what here? `cloud.buildbuddy.io` sans the scheme?

Reply:
yeah, checking for the string `"cloud.buildbuddy.io"`, scheme-less. I think if it's configured that way, we'll accept it and it'll default to grpcs://, but I'm not totally sure